Makefile

```diff
@@ -234,7 +252,7 @@ bundle-publish:
 	&& cd ${DIR} \
 	&& FULL_IMAGE_NAME_PREFIX=${FULL_IMAGE_NAME_PREFIX} \
 		${MAKEFILE_DIR}/devops/scripts/bundle_runtime_image_push.sh \
-	&& porter publish --registry "$${ACR_NAME}.azurecr.io" --debug
+	&& porter publish --registry "$${ACR_NAME}.azurecr.io" --force
```

Comment:
Is the use of `--force` here to overwrite a bundle version if it already exists in the registry?

Reply:
Yes. Porter now checks if the version is already in the registry and fail the publish action.